templates/support/community-support.html

```diff
@@ -29,8 +29,8 @@ <h1>Community support</h1>
 <div class="p-strip is-deep">
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <h3><a href="https://discourse.ubuntu.com/t/community-support/709">Find community support</a></h3>
-      <p><a href="https://askubuntu.com/">Ask Ubuntu</a> and <a href="https://ubuntuforums.org/">Ubuntu Forums</a> are great sources of answers to common questions about installing, troubleshooting and optimising Ubuntu. For more info on support forums, including IRC channels, see <a href="https://discourse.ubuntu.com/t/community-support/709">our Discourse site</a>.</p>
+      <h3><a href="/community/support">Find community support</a></h3>
+      <p><a href="https://askubuntu.com/">Ask Ubuntu</a> and <a href="https://ubuntuforums.org/">Ubuntu Forums</a> are great sources of answers to common questions about installing, troubleshooting and optimising Ubuntu. For more info on support forums, including IRC channels, see <a href="/community/support">our Discourse site</a>.</p>
```

Comment:
```suggestion
      <p><a href="https://askubuntu.com/">Ask Ubuntu</a> and <a href="https://ubuntuforums.org/">Ubuntu Forums</a> are great sources of answers to common questions about installing, troubleshooting and optimising Ubuntu. For more info on support forums, including IRC channels, see <a href="/community/support">our community docs</a>.</p>
```
Thanks for the changes @aaronprisk. The text for this link needs to change as well since it is no longer leading users to discourse. Please accept this suggestion or update with your preferred copy and we'll merge